Makefile

```
@@ -53,18 +56,31 @@ FINCH_OS_IMAGE_INSTALLATION_LOCATION ?= $(DEST)/os/$(FINCH_OS_BASENAME)
 all: binaries
 
 .PHONY: binaries
+.PHONY: download
+
+# Rootfs required for Windows, require full OS for Linux and Mac
+ifeq ($(OS), Windows_NT)
```

Not sure if we want to gate it like this. Wouldn't that make it harder to do cross-builds?

---

good point. how about something like this?

```suggestion
BUILD_OS ?= "$(OS)"
ifeq ($(BUILD_OS), Windows_NT)
```

so that we can write over `BUILD_OS` aka `BUILD_OS=Darwin make`